x-pack/plugins/security_solution/public/overview/pages/overview.tsx

```diff
@@ -66,87 +69,91 @@ const OverviewComponent: React.FC<PropsFromRedux> = ({
     addMessage('management', 'dismissEndpointNotice');
   }, [addMessage]);
   const { allEnabled: isIngestEnabled } = useIngestEnabledCheck();
-
-  return (
+  return indicesExist !== false ? (
     <>
-      {indicesExist ? (
-        <StickyContainer>
-          <FiltersGlobal>
-            <SiemSearchBar id="global" indexPattern={indexPattern} />
-          </FiltersGlobal>
-
-          <WrapperPage>
-            {!dismissMessage && !metadataIndexExists && isIngestEnabled && (
-              <>
-                <EndpointNotice onDismiss={dismissEndpointNotice} />
-                <EuiSpacer size="l" />
-              </>
-            )}
-            <EuiFlexGroup gutterSize="none" justifyContent="spaceBetween">
-              <SidebarFlexItem grow={false}>
-                <StatefulSidebar />
-              </SidebarFlexItem>
-
-              <EuiFlexItem grow={true}>
-                <EuiFlexGroup direction="column" gutterSize="none">
-                  <EuiFlexItem grow={false}>
-                    <SignalsByCategory
-                      filters={filters}
-                      from={from}
-                      indexPattern={indexPattern}
-                      query={query}
-                      setAbsoluteRangeDatePicker={setAbsoluteRangeDatePicker}
-                      setQuery={setQuery}
-                      to={to}
-                    />
-                    <EuiSpacer size="l" />
-                  </EuiFlexItem>
-
+      <StickyContainer>
+        <FiltersGlobal>
+          <SiemSearchBar id="global" indexPattern={indexPattern} />
+        </FiltersGlobal>
+
+        <WrapperPage>
+          {!dismissMessage && !metadataIndexExists && isIngestEnabled && (
+            <>
+              <EndpointNotice onDismiss={dismissEndpointNotice} />
+              <EuiSpacer size="l" />
+            </>
+          )}
+          <EuiFlexGroup gutterSize="none" justifyContent="spaceBetween">
+            <SidebarFlexItem grow={false}>
+              {indicesExist !== true ? (
```

Comment:
Put `StatefulSidebar ` behind a loader to fix this error:
<img width="1667" alt="Screenshot 2020-07-23 at 12 55 42" src="https://user-images.githubusercontent.com/6295984/88295743-bbeac300-ccf5-11ea-9456-37c0faa49e5f.png">